appdata.patch

```diff
@@ -0,0 +1,11 @@
+--- a/org.kde.minuet.appdata.xml	2020-07-21 21:25:43.538444000 +0700
++++ b/org.kde.minuet.appdata.xml	2020-07-22 16:31:28.870236846 +0700
+@@ -126,4 +126,8 @@
+   <provides>
+     <binary>minuet</binary>
+   </provides>
++  <releases>
++    <release version="20.04.3" date="2020-07-07"/>
++  </releases>
++  <content_rating type="oars-1.1" />
```

Comment:
would you please submit a merge request with the oars part upstream? 

The version part is trickier, since minuet calls itself 0.4.0 but has been doing so for a longer time, so i need to do some convincing (or waiting in the emptynes of no answers) on whether to move enterelly to the year.month naming or at least increase that 0.4.x periodically

Reply:
> would you please submit a merge request with the oars part upstream?

https://invent.kde.org/education/minuet/-/merge_requests/14

Reply:
> The version part is trickier, since minuet calls itself 0.4.0 but has been doing so for a longer time, so i need to do some convincing (or waiting in the emptynes of no answers) on whether to move enterelly to the year.month naming or at least increase that 0.4.x periodically

Should we use 0.4.0 or 20.04.3?

Reply:
I'd go with 0.4.0 which is what the code calls itself